Fix error for non existing docker (#864)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
* Fixed an issue in the **openapi_codegen** command where it created duplicate functions name from the swagger file.
* Fixed an issue in the **update-release-notes** command where the *update type* argument was not verified.

> * Fixed an issue in the **validate** command where a non-existing docker image presented and no error was showed.

Reviewer: ```suggestion
* Fixed an issue in the **validate** command where if a non-existing docker image was presented, no error was raised.
```
